### PR TITLE
Turn on remote rm_test 

### PR DESCRIPTION
**Pull request by QiWang19**
Turn on remote rm_test --cidfile

Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: pkg/domain/infra/tunnel/containers.go [hunk 1]
```diff
@@ -162,6 +163,14 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 	var (
 		reports []*entities.RmReport
 	)
+	for _, cidFile := range options.CIDFiles {
+		content, err := ioutil.ReadFile(cidFile)
+		if err != nil {
+			return nil, errors.Wrap(err, "error reading CIDFile")
```
Comment (mheon) [inline pkg/domain/infra/tunnel/containers.go:169]: Can you put the path of the CID file in the error?

Comment (QiWang19) [reply pkg/domain/infra/tunnel/containers.go]: fixed

**Pull request merged**